fastlane-plugin-wpmreleasetoolkit.gemspec

```diff
@@ -12,7 +12,7 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/wordpress-mobile/release-toolkit'
   spec.license       = 'MIT'
 
-  spec.files         = Dir['lib/**/*'] + %w[README.md LICENSE]
+  spec.files         = Dir['lib/**/*'] + Dir['ext/drawText/{makefile.example,drawText/**/*,drawText Tests/**/*,drawText.xcodeproj/**/*}'] + %w[README.md LICENSE]
```

Comment:
I decided to specify the list of files in `ext/drawText` explicity (rather than using a more generic `ext/drawText/**/*`) because there could be uncommited files in that directory that we would otherwise accidentally include when running `gem build` on a dirty repo / working copy.

This was actually the case on my Mac since I also had a `test-output/` folder in `ext/drawText`

I also had to include `drawText Tests` even though they're not really useful to build the binary, but they are referenced by the `xcodeproj` so I'm assuming that not having them included in the `.gem` would make `xcodebuild` complain that some files in the xcodeproj could not be found; better safe that sorry.